lib/helpers/is-route-template.js

```diff
@@ -19,6 +19,6 @@ module.exports = function isRouteTemplate(moduleName) {
   var isPartial = baseName.indexOf('-') === 0;
 
   return (
-    !isPartial && moduleName.indexOf('components/') === -1 // classic component
+    !isPartial && moduleName.indexOf('templates/components/') === -1 // classic component
```

Comment:
To be even safer, could you change it to `/templates/components/` with the leading slash?

Reply:
sure, done